### ISSUE TITLE
some questions about training

### ISSUE THREAD

**Issue by hdasd** [cvlab-stonybrook/DM-Count#18]
Hello! When I was trying to replicate the results about ShanghaiTech A. The best result is Mae = 61.7. This is two points different from your experimental results. When I read your article, I found that you set λ 1 to 0.1 and λ 2 to 0.01. Does this mean that in the process of training, the weight of loss is different, loss = 0.1 * ot_ loss + count_ loss +0.01* tv_ loss？ Could you explain it to me? thank you!

**Comment by Boyu-Wang** [comment]
That's correct. \labmda_1 is for OT loss, \labmda_2 is for TV loss. 
Did you change the data loader for Sh part A? Dataloader could affect the performance on Sh part A. 

Attached is the training log file for Sh part A.
[sha-train.log](https://github.com/cvlab-stonybrook/DM-Count/files/6103962/sha-train.log)

**Comment by Boyu-Wang** [comment]
Update in #19.
